package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "theforeman",
+  "version": "1.11.0",
+  "license": "GPL-3.0",
+  "description": "Full lifecycle management system",
```

Comment:
sounds reasonable, if we ever introduce node, we should probably split the
packages anyway.

On Tue, Jun 14, 2016 at 2:59 PM, Tomer Brisker <notifications@github.com>
wrote:

> In package.json
> <https://github.com/theforeman/foreman/pull/3433#discussion_r66955770>:
>
> > @@ -0,0 +1,25 @@
> > +{
> > +  "name": "theforeman",
> > +  "version": "1.11.0",
> > +  "license": "GPL-3.0",
> > +  "description": "Full lifecycle management system",
>
> I wonder if maybe we should do something like cockpit does here:
> https://github.com/cockpit-project/cockpit/blob/master/package.json since
> we aren't really a node module
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/theforeman/foreman/pull/3433/files/8f53a9c9d7358b0b91e136bc2010b1d69fd9fadd#r66955770>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AABOx63pVrwCzr-lEJeveBzZwLEiu_ryks5qLpe9gaJpZM4IJHWI>
> .
>